entity.go

```diff
@@ -57,3 +55,13 @@ func (e *Entity) SetPosition(x, y int) {
 func (e *Entity) SetCell(x, y int, c *Cell) {
 	renderCell(&e.canvas[x][y], c)
 }
+
+// Fill fills the canvas of the Entity with
+// a c Cell.
```

Comment:
This should be

```go
// a Cell c.
```

(Sorry to be picky but I think we need to keep the GoDoc as perfect as possible!)